engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/JsForeignNode.java

```diff
@@ -0,0 +1,99 @@
+package org.enso.interpreter.epb;
+
+import com.oracle.truffle.api.dsl.NodeField;
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.interop.InteropException;
+import com.oracle.truffle.api.interop.InteropLibrary;
+import com.oracle.truffle.api.interop.InvalidArrayIndexException;
+import com.oracle.truffle.api.interop.TruffleObject;
+import com.oracle.truffle.api.interop.UnsupportedMessageException;
+import com.oracle.truffle.api.library.CachedLibrary;
+import com.oracle.truffle.api.library.ExportLibrary;
+import com.oracle.truffle.api.library.ExportMessage;
+import java.util.Arrays;
+
+/** A node responsible for performing foreign JS calls. */
+@NodeField(name = "foreignFunction", type = Object.class)
+@NodeField(name = "arity", type = int.class)
```

Comment:
This NodeField is not used anywhere?

Reply:
It was used [here](https://github.com/enso-org/enso/pull/7882/files/3fa992a7e115ee6f0e3f64af3c52de353448aef5#diff-a2e3756dfa68e081d43ee49566f83821651587e059be6bac47bf1383351b2e25L37).

Good catch. Removed in 3f63d9ec04